transform.go

```diff
@@ -79,19 +80,23 @@ func opTypeToString(value OpType) string {
 	case OpTypeIn:
 		return "in"
 	}
-
 	return "??unknown??"
 }
 
 type OpNode struct {
 	BucketIdx BucketID
 	Op        OpType
+	OpNot     bool
 	Rhs       interface{}
 }
 
 func (op OpNode) String() string {
 	var out string
-	out += fmt.Sprintf("[%d] %s", op.BucketIdx, opTypeToString(op.Op))
+	var opNotStr string
```

Comment:
This is from the old implementation of NotExpr, which is already completed

Reply:
Removed